core/src/main/java/pl/agh/edu/utils/ScenariosAdapter.java

```diff
@@ -0,0 +1,23 @@
+package pl.agh.edu.utils;
+
+import pl.agh.edu.enums.DifficultyLevel;
+import pl.agh.edu.enums.HotelType;
+import pl.agh.edu.screen.ScenariosScreen;
+
+public class ScenariosAdapter {
+	private HotelType hotelType;
+	private DifficultyLevel difficultyLevel;
+
+	public void extractDataFromScenariosScreen(ScenariosScreen screen) {
```

Comment:
ja bym przez konstruktor przekazał hotelType i difficulty a nie screen z którego je wyciągamy.

Reply:
z tego co widze to ta metoda jest uzywana raz - gdy klikamy play. mozna wiec gdy klikniemy play zainicjalizować SenariosAdapter